Source/JavaScriptCore/runtime/IntlObjectInlines.h

```diff
@@ -167,25 +167,24 @@ ResultType intlStringOrBooleanOption(JSGlobalObject* globalObject, JSObject* opt
     JSValue value = options->get(globalObject, property);
     RETURN_IF_EXCEPTION(scope, { });
 
-    if (!value.isUndefined()) {
-        if (value.isBoolean() && value.asBoolean())
-            return trueValue;
+    if (value.isUndefined())
+        return fallback;
 
-        bool valueBoolean = value.toBoolean(globalObject);
-        RETURN_IF_EXCEPTION(scope, { });
+    if (value.isBoolean() && value.asBoolean())
+        return trueValue;
```

Comment:
Shouldn't we handle explicit `false` here too?
```suggestion
    if (value.isBoolean())
        return value.asBoolean() ? trueValue : falsyValue;
```

Reply:
Oops! Correct. Fix, and add tests.

Reply:
Ah, no. According to the spec change, we are delegating this evaluation to `if (!valueBoolean)`.

```
1. If _value_ is *undefined*, return _fallback_.
1. If _value_ is *true*, return _trueValue_.
1. Let _valueBoolean_ be ToBoolean(_value_).
1. If _valueBoolean_ is *false*, return _falsyValue_.
1. Let _value_ be ? ToString(_value_).
1. If _values_ does not contain an element equal to _value_, return _fallback_.
1. Return _value_.
```

Reply:
Either way should be correct, I just thought it might be a little nicer as code. 😅

Reply:
Ah, I see. I think probably the current code is fine since it is well aligned to the spec text ;)